eo-parser/src/main/java/org/eolang/parser/ParsingTrain.java

```diff
@@ -103,9 +104,17 @@ public ParsingTrain() {
                             shift,
                             new StLambda(
                                 shift::uid,
-                                (pos, xml) -> ParsingTrain.EACH.with("step", pos)
-                                    .with("sheet", shift.uid())
-                                    .transform(xml)
+                                (pos, xml) -> {
```

Comment:
@MikhailLipanin I think we should throw an exception if `critical` is seen. You just quietly ignore this fact now. I suggest we create two lambdas. One will do what this `EACH` is doing now, second one will check for severity presence. See: https://github.com/yegor256/xsline/issues/42 (you are welcome to implement it)

Reply:
@yegor, let's figure out, what for we will throw an Exception.
How it works now:
```
if (failOnError == TRUE) {
  Exception throws in Sanitized.java if there are "error"s in list of <errors>
  Exception throws in XSL-s due to <xsl:message terminate="yes"> at some XSL-s
} else {
  Exception throws in XSL-s due to <xsl:message terminate="yes"> at some XSL-s
}
```
if we make, that exception will throws when we find "critical" error, behavior of exceptions won't be different from now - they will also be thrown as at scheme above.
So may be we can just stop a Pipeline, how it works now at this PullReques, so at Transpile phase it will be shown, that there is a "critical" error and it is only one ?

Reply:
The example of throwing XSL-exception is in ticket #1024